build.rs

```diff
@@ -5,7 +5,7 @@ use std::process::{Command, Stdio};
 use std::{env, fs::*, io::Write, path::PathBuf};
 
 fn compute_static_hash() -> String {
-    //"find static/ -type f ! -path 'static/media/*' | sort | xargs stat --printf='%n %Y\n' | sha256sum"
+    //"find static/ -type f ! -path 'static/media/*' | sort | xargs stat --printf='%n %Y\n' | openssl dgst -r"
```

Comment:
i'd prefer we leave `stat(1)` out of this, as the `--printf` option is not platform independent, and no equivalent, platform independent option exists

the only other would be to use `-c`

Reply:
The goal of this function is to have a unique id, and best effort is made to not invalidate cache if not necessary. We can also not make that best effort (recompiling Plume => new cache id) and just generate a random hex string. That would be platform independent.
What shall I do?

Reply:
i think i'd just go with `stat -c` for now

there's not a big chance we'll be building on windows… easily… i think, so let's just go with this.